### DISCUSSION TITLE
Will a message box be added in the future?

### DISCUSSION THREAD

**Discussion by M737** [Textualize/textual#1252] [Q&A] [answered]
message box， which confirm yes or no, even input some text

will  this widget  be added in the future?

**Comment by davep** [comment, marked as answer]
I don't believe there's any specific plans to add such a thing, at least not at the moment, and I don't see any mention of something like that [on the roadmap](https://textual.textualize.io/roadmap/) -- that's not to say it won't get added at some point. What I have seen so far is folk using what Textual provides at the moment to make such things.
